region/wreckedship/main.json

```diff
@@ -2854,8 +2910,33 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": [],
-                  "devNote": "No alternate strat since the only tricky jump is easy on the way back."
+                  "requires": [
+                    {"or":[
+                      "f_DefeatedPhantoon",
+                      "SpaceJump",
+                      {"thornHits": 2},
+                      {"and":[
+                        {"thornHits": 1},
+                        {"or":[
+                          "HiJump",
+                          "canWalljump",
+                          "canIframeSpikeJump",
+                          "Grapple"
```

Comment:
How does the grapple help?  Without a precise fling.

Reply:
A mediocre fling will put you far enough to the right to be able to only take 1 spike hit, even without optimizing iframes

Reply:
I think the expected fling takes you as far as jumping from the ledge.
`canDamageBoost` could be added too.
Or XMode lol.

Reply:
- Added dboost
- I can grapple further than I can jump, fairly easily. The distance is enough that you can immediately jump again and be out of the spikes. 
- I know it was a joke, but Xmode would likely assume iframe spike jump, so I think it isnt needed here

Reply:
I think getting a useful fling is a tech and default grappling is a swing and release.  It's possible that can be an assumed tech; I'm interested in adding more assumed techs like `canJumpMorph`.  In this case there are many intuitive ways to play the room to avoid the second hit that it hardly matters.

Reply:
For XMode requirements would look like; `canCarefulJump`, `Morph`, and `canOffScreenMovement`.  It's a frame perfect unmorph on spikes but that's your only interaction with the spikes.  The bounce can be done on the upper ledge by the Etank chozo.